bql/grammar/grammar.go

```diff
@@ -815,6 +818,17 @@ func BQL() *Grammar {
 				},
 			},
 		},
+		"REIFICATION_CLAUSE": []*Clause{
+			{
+				Elements: []Element{
+					NewTokenType(lexer.ItemSemicolon),
+					NewSymbol("CONSTRUCT_PREDICATE"),
+					NewSymbol("CONSTRUCT_OBJECT"),
+					NewSymbol("REIFICATION_CLAUSE"),
```

Comment:
Currently, this allows multiple levels of indirection for reification purposes. Should we allow that?